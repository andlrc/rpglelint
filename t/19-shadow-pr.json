[
   {
      "column":9,
      "file":"t/19-shadow-pr.rpgle",
      "line":6,
      "message":"declaration of 'pr1' shadows a global declaration",
      "severity":"warning"
   },
   {
      "column":8,
      "file":"t/19-shadow-pr.rpgle",
      "line":2,
      "message":"shadowed declaration is here",
      "severity":"note"
   }
]
