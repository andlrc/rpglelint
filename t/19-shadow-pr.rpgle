// rpglelint: -Wshadow
dcl-pr pr1;
end-pr;

dcl-proc a;
  dcl-s pr1 int(10);
end-proc;
