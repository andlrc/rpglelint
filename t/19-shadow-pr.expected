t/19-shadow-pr.rpgle:6:9: warning: declaration of 'pr1' shadows a global declaration [-Wshadow]
   dcl-s pr1 int(10);
         ^
t/19-shadow-pr.rpgle:2:8: note: shadowed declaration is here
 dcl-pr pr1;
        ^
